<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>560</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Memento-french</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterPlayerSubtitles">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="MpvWidget" name="player">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="SubtitleListWidget" name="subtitleList" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="PlayerControls" name="controls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMedia">
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenUrl"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Audio</string>
    </property>
    <addaction name="actionAudioNone"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionVideoNone"/>
   </widget>
   <widget class="QMenu" name="menuSubtitle">
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Subtitle</string>
    </property>
    <widget class="QMenu" name="menuSubtitleTwo">
     <property name="title">
      <string>Second Track</string>
     </property>
     <addaction name="actionSubtitleTwoNone"/>
    </widget>
    <addaction name="actionAddSubtitle"/>
    <addaction name="menuSubtitleTwo"/>
    <addaction name="separator"/>
    <addaction name="actionSubtitleNone"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="geometry">
     <rect>
      <x>1003</x>
      <y>325</y>
      <width>200</width>
      <height>184</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuAnkiProfile">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>189</y>
       <width>150</width>
       <height>52</height>
      </rect>
     </property>
     <property name="title">
      <string>Anki Profile</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuSubtitle_Search">
     <property name="title">
      <string>Subtitle Search</string>
     </property>
     <addaction name="actionIncreaseSize"/>
     <addaction name="actionDecreaseSize"/>
     <addaction name="actionMoveUp"/>
     <addaction name="actionMoveDown"/>
    </widget>
    <addaction name="menuAnkiProfile"/>
    <addaction name="menuSubtitle_Search"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuMedia"/>
   <addaction name="menuAudio"/>
   <addaction name="menuVideo"/>
   <addaction name="menuSubtitle"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAudioNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionVideoNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSubtitleNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSubtitleTwoNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionAddSubtitle">
   <property name="text">
    <string>Add Subtitle File</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Memento</string>
   </property>
  </action>
  <action name="actionIncreaseSize">
   <property name="text">
    <string>Increase Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionDecreaseSize">
   <property name="text">
    <string>Decrease Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="text">
    <string>Open URL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MpvWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>src/gui/widgets/mpvwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PlayerControls</class>
   <extends>QWidget</extends>
   <header>src/gui/widgets/playercontrols.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubtitleListWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/widgets/subtitlelistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
